lib/rex/post/meterpreter/extensions/stdapi/fs/dir.rb

```diff
@@ -194,9 +195,9 @@ def Dir.unlink(path)
   # Downloads the contents of a remote directory a
   # local directory, optionally in a recursive fashion.
   #
-  def Dir.download(dst, src, recursive = false, force = true, &stat)
+  def Dir.download(dst, src, recursive = false, force = true, glob = nil, &stat)
```

Comment:
mmm I think the line 234 (the recursive call) should be changed still since we're not using keyword arguments here:

```
download(dst_item, src_item, recursive, force, &stat)
```

should become

```
download(dst_item, src_item, recursive, force, glob, &stat)
```

Also adding new parameters at the end uses to mitigate changes in the method signature I think.